docs/admin/node-allocatable.md

```diff
@@ -0,0 +1,144 @@
+This is an issue because nodes typically run quite a few system daemons that power the OS and Kubernetes itself.
+Unless resources are set aside for these system daemons, pods and system daemons will compete for resources and lead to resource starvation issues on the node.
+The `kubelet` exposes a feature named `Node Allocatable` that helps to reserve compute resources for system daemons.
+Kubernetes recommends cluster administrators to configure `Node Allocatable` based on their workload density on each node.
+
+## Node Allocatable
+
+      Node Capacity
+---------------------------
+|     kube-reserved		  |
+|-------------------------|
+|	  system-reserved	  |
+|-------------------------|
+|    eviction-threshold	  |
+|-------------------------|
+|						  |
+|  		allocatable		  |
+|  	    (available        |
+|  	   	 for pods)     	  |
+|  						  |
+|						  |
+---------------------------
+
+`Allocatable` on a Kubernetes node is defined as the amount of compute resources that are available for pods.
+The scheduler does not over subscribe `allocatable`.
+`CPU` and `memory` are supported as of now.
+Support for `storage` will be added in the future.
+
+Node Allocatable is exposed as part of `v1.Node` object in the API and as part of `kubectl describe node` in the CLI.
+
+Resources can be reserved for two categories of system daemons in the `kubelet`.
+
+### Kube Reserved
+
+**Kubelet Flag**: `--kube-reserved=[cpu=100mi][,][memory=100Mi]`
+**Kubelet Flag**: `--kube-reserved-cgroup=`/runtime.slice`
+
+`kube-reserved` is meant to capture resource reservation for kubernetes system daemons like the `kubelet`, `container runtime`, `node problem detector`, etc.
+It is not meant to reserve resources for system daemons that are run as pods.
+`kube-reserved` is typically a function of `pod density` on the nodes.
+[This performance dashboard](http://node-perf-dash.k8s.io/#/builds) exposes `cpu` and `memory` usage profiles of `kubelet` and `docker engine` at multiple levels of pod density.
+[This blog post](http://blog.kubernetes.io/2016/11/visualize-kubelet-performance-with-node-dashboard.html) explains how the dashboard can be interpreted to come up with a suitable `kube-reserved` reservation.
+
+It is recommended that the kubernetes system daemons are placed under a top level control group (`system.slice` on systemd machines for example).
+Each system daemon should ideally run within its own child control group.
+Refer to [this doc](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/node-allocatable.md#recommended-cgroups-setup) for more details on recommended control group hierarchy.
+
+To optionally enforce `kube-reserved` on system daemons, specify the parent control group for kube daemons as the value for `--kube-reserved-cgroup` kubelet flag.
+
+### System Reserved
+
+**Kubelet Flag**: `--system-reserved=[cpu=100mi][,][memory=100Mi]`
+**Kubelet Flag**: `--system-reserved-cgroup=`/system.slice`
```

Comment:
we need to make clear that the kubelet doesnt create either of these two cgroups.